### PR TITLE
Add support for a catch-all "*" file in `regenerate_comments`

### PR DESCRIPTION
**Pull request by gkoz**

### CODE REVIEW

File: src/regenerate.rs [hunk 1]
```diff
@@ -86,110 +87,121 @@ fn get_corresponding_type(elements: &[(Option<TypeStruct>, Vec<String>)],
...
+}
 
-            // first, we need to put back file comment
-            for entry in elements.iter() {
-                if entry.0.is_none() {
-                    let mut it = 0;
+fn do_regenerate(path: &str, parse_result: &mut ParseResult,
+                 elements: &mut Vec<(Option<TypeStruct>, Vec<String>)>) {
+    let mut position = 0;
+    let mut decal = 0;
```
Comment (GuillaumeGomez) [inline src/regenerate.rs]: So you removed file indent also and regenerate it here?

**Comment by GuillaumeGomez** [comment]
Can you also add an explanation somewhere of the meaning of '?' please?

**Comment by gkoz** [comment]
When documenting this, `"*"` became more logical to me.. ;)

File: src/regenerate.rs [hunk 2]
```diff
@@ -236,7 +251,9 @@ fn save_remainings(infos: &HashMap<String, Vec<(Option<TypeStruct>, Vec<String>)
                 if content.len() < 1 {
                     continue;
                 }
-                let _ = writeln!(out_file, "{}{}", FILE_COMMENT, key);
+                // Set the name to "*" for entries that ignore file name
+                let key = key.as_ref().map(|s| &s[..]).unwrap_or("*");
+                let _ = writeln!(out_file, "{}{}", FILE, key);
```
Comment (gkoz) [inline src/regenerate.rs]: I thinks the intention here was to output `FILE` not `FILE_COMMENT`.

Comment (GuillaumeGomez) [inline src/regenerate.rs]: Oh indeed. Thanks!

**Comment by GuillaumeGomez** [comment]
'*' instead of '?'. That seems more understandable at least. ;)

**Comment by gkoz** [comment]
Heh, I thought this had long been merged :snail:

**Comment by GuillaumeGomez** [comment]
Me too, my bad!

**Pull request merged**